WebUI/generate.sh

```diff
@@ -41,9 +42,9 @@ BUILD_DIR=$(pwd)/yaydoctemp
 
 cd $ROOT_DIR
 
-cp -a ../../../scripts/ $BUILD_DIR/
-cp -a ../../../templates/ $BUILD_DIR/
-cp -a ../../../requirements.txt $BUILD_DIR/
+cp -a ../../../../scripts/ $BUILD_DIR/
+cp -a ../../../../templates/ $BUILD_DIR/
+cp -a ../../../../requirements.txt $BUILD_DIR/
```

Comment:
in order to copy it has to go one more depth. previously it was going upto `WebUI`, so i have added one more depth